scripts/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -115,18 +115,28 @@ def extract_b2b_params(workspace):
         return [params_north, params_south]
 
     def load_existing_calibration_files(self, file_path):
+        import pydevd_pycharm
+        pydevd_pycharm.settrace(stdoutToServer = True, stderrToServer = True)
```

Comment:
Looks like you left this debugging line in by accident!

Reply:
Good spot! I've just removed it